src/editor/components/modals/revision-list.jsx

```diff
@@ -23,6 +23,7 @@ class RevisionListUnconnected extends React.Component {
       })
     ),
     selectedRevisionId: PropTypes.number,
+    hideLocalChanges: PropTypes.bool,
```

Comment:
I think this would be more intuitive to express as `haveLocalChanges`, rather than its negation.

Reply:
Makes sense